cluster-autoscaler/config/crd/provisioning.k8s.io_provisioningrequests.yaml

```diff
@@ -0,0 +1,203 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    api-approved.kubernetes.io: https://github.com/kubernetes/autoscaler/pull/5848
+    controller-gen.kubebuilder.io/version: v0.13.0
+  name: provisioningrequests.provisioning.k8s.io
```

Comment:
The `provisioning.k8s.io` API group looks novel. What else would go in this group, would it only contain autoscaling APIs?